Tasks/PublishCodeCoverageResultsV2/task.json

```diff
@@ -0,0 +1,61 @@
+{
+    "id": "2A7EBC54-C13E-490E-81A5-D7561AB7CD97",
+    "name": "PublishCodeCoverageResults",
+    "friendlyName": "Publish code coverage results",
+    "description": "Publish any of the  code coverage results supported by report generator from a build",
+    "helpUrl": "https://docs.microsoft.com/azure/devops/pipelines/tasks/test/publish-code-coverage-results",
+    "helpMarkDown": "[Learn more about this task](https://go.microsoft.com/fwlink/?LinkID=626485)",
+    "category": "Test",
+    "visibility": [
+        "Build"
+    ],
+    "runsOn": [
+        "Agent",
+        "DeploymentGroup"
```

Comment:
There isn't a need to have this run on a deployment group. We should remove that.